src/navigation/AppNavigation.js

```diff
@@ -10,6 +10,7 @@ import CreateEvaluationScreen from '../screens/CreateEvaluation';
 import GenericFormDemoScreen from '../screens/GenericFormDemo';
 import AnnouncementsScreen from '../screens/Announcements';
 import AssignmentListScreen from '../screens/AssignmentList';
+import { ClassroomHome, CreateClassroomScreen } from '../screens/Classroom';
```

Comment:
What did you have in mind for the ClassroomHome Screen? I'm thinking in the final implementation, when the user opens the app, the Announcements/Assignments Screen is the "Home Screen", and (for now) the teacher can modify classroom details in Settings or something. The classroom name will be displayed somewhere else (the designers and I will need to work on that because both teachers and students will be able to be part of multiple classrooms - ie. a teacher can teach multiple classes and a student can be part of "Beginning Orchestra" and "Beginning Cello")

Reply:
Sounds good - ClassroomHome was more of a placeholder so I can make sure my backend connections were working. I think most of the code can be moved around when/if necessary.